### ISSUE TITLE
How to set local path phantomjs 2 bin ?

### ISSUE THREAD

**Issue by Fost** [karma-runner/karma-phantomjs-launcher#80]
Hi !

How I can set local path to phatom 2 ? On win/linux/darwin.
Without download every time.

Thanks !

**Comment by grzegorz-dubicki** [comment]
Set PHANTOMJS_BIN env to PhantomJs bin:

```
22:07:29 grzegorz@myhost:~/git/some-dir$ phantomjs -v
2.0.0
22:08:14 grzegorz@myhost:~/git/some-dir$ which phantomjs
/usr/local/bin/phantomjs
22:08:20 grzegorz@myhost:~/git/some-dir$ export PHANTOMJS_BIN=`which phantomjs`
22:08:42 grzegorz@myhost:~/git/some-dir$ gulp test
[22:08:49] Using gulpfile ~/git/some-dir/gulpfile.js
[22:08:49] Starting 'test'...
05 09 2015 22:08:49.292:INFO [karma]: Karma v0.13.9 server started at http://localhost:9018/
05 09 2015 22:08:49.298:INFO [launcher]: Starting browser PhantomJS
05 09 2015 22:08:50.740:INFO [PhantomJS 2.0.0 (Mac OS X 0.0.0)]: Connected on socket YAcKCXTDvg5w64PQAAAA with id 43811757
(...)
```

**Comment by bretkikehara** [comment]
@grzegorz-dubicki Thanks for the fix!

I had ran into this issue on [TravisCI's Ubuntu Trusty beta image](http://blog.travis-ci.com/2015-10-14-opening-up-ubuntu-trusty-beta/). With the latest build images, the directory to PhantomJS was changed.

### Error
```bash
INFO [karma]: Karma v0.12.37 server started at http://localhost:9876/
INFO [launcher]: Starting browser PhantomJS
ERROR [launcher]: Cannot start PhantomJS
	Can not find the binary /usr/local/phantomjs/bin/phantomjs
	Please set env variable PHANTOMJS_BIN
```

### Solution
```bash
export "PHANTOMJS_BIN=`which phantomjs`"
```

**Comment by unional** [comment]
Tried to do that on windows, but couldn't get it to work. Karma stuck at `Starting browser PhantomJS`.
`set PHANTOMJS_BIN=e:\phantomjs-2.0.0\bin`
`set PHANTOMJS_BIN=e:\phantomjs-2.0.0\bin\phantomjs.exe`

**Comment by dignifiedquire** [comment]
The newly released 1.0.0 is able to use the latest phantomjs-prebuilt which brings phantomjs@2 easily into your setup.
